docker-compose.yml

```diff
@@ -13,9 +13,6 @@ services:
     tty: true
     command: ["/bin/zsh"]
     volumes:
-      - ${HOME}/.gitconfig:/home/ubuntu/.gitconfig
-      - ${HOME}/.ssh:/home/ubuntu/.ssh
-      - ${PWD}:/app
```

Comment:
This is for local credentials and allows development entirely within the safety of the container.